### PR TITLE
hint and and link to set operators from Range

### PR DESCRIPTION
**Pull request by gfldex**

### CODE REVIEW

**Comment by JJ** [comment]
It would have helped if you had said something about the problem (issue, for instance) and what the solution is about. I'll check it anyway.

File: doc/Type/Range.pod6 [hunk 1]
```diff
@@ -69,6 +69,15 @@ It is possible to shift or scale the interval of a range:
     say (1..10) - 1;       # OUTPUT: «0..9␤»
     say (1..10) * 2;       # OUTPUT: «2..20␤»
     say (1..10) / 2;       # OUTPUT: «0.5..5.0␤»
+    
+=head2 Matching against Ranges
+
+Use L<set operators|/language/setbagmix> to match against Ranges. Please note
+that set operators may cast to L<Set|/type/Set> and can create heavy memory load.
```
Comment (JJ) [inline doc/Type/Range.pod6]: Maybe clarify when that would happen? Does it happen always? Why would that create heavy load?

**Comment by gfldex** [comment]
forget it, the docs are all yours

**Comment by JJ** [comment]
Hey, no, they are not... It's just a small change...

**Pull request merged**